app/src/main/java/ece/wisc/opencvtest/EyeDetection.java

```diff
@@ -519,6 +529,21 @@ public void relearnFace(View v)
 
     public void setOffsets(double[][] offsets) {
         this.offsets = offsets;
-        this.sendBT = true;
+    }
+
+    public int getMouse_x() {
```

Comment:
I don't think we ended up actually using these? maybe we should be or get rid of them?

Reply:
They could be deleted, for now it's fine if they aren't used